packages/uikit/src/components/desktop/multi-send/MultiSendConfirmNotification.tsx

```diff
@@ -0,0 +1,286 @@
+
+        return { rows };
+    }, [form.rows, bnAmounts, asset.decimals]);
+
+    const { fiat } = useAppContext();
+    const willBeSentInFiat = formatFiatCurrency(
+        fiat,
+        willBeSentBN?.multipliedBy(rate?.prices || 0)
+    );
+
+    const { account } = useAppContext();
+    const { data: wallet } = useWalletState(account.activePublicKey!);
+
+    const {
+        mutateAsync: estimate,
+        isLoading: estimateLoading,
+        data: estimateData,
+        error: estimateError
+    } = useEstimateMultiTransfer();
+
+    useEffect(() => {
+        estimate({ form: formTokenized, asset }).catch(() => setTimeout(onClose, 5000));
+    }, [asset, formTokenized]);
+
+    const tonFee = estimateData?.fee.stringAssetRelativeAmount;
+    const fiatFee = formatFiatCurrency(
+        fiat,
+        estimateData?.fee.relativeAmount.multipliedBy(tonRate?.prices || 0) || new BigNumber(0)
+    );
+
+    return (
+        <ConfirmWrapper>
+            {image ? <Image full src={image} /> : <ImageMock full />}
+            <TransferLabel>Transfer</TransferLabel>
+            <Num2>{willBeSent}</Num2>
+            <FiatValue>{willBeSentInFiat}</FiatValue>
+            <ListBlockStyled noUserSelect>
+                <ListItemStyled hover={false}>
+                    <Body2>Wallet</Body2>
+                    {wallet && (
+                        <WalletNameStyled>
+                            <WalletEmoji
+                                emojiSize="16px"
+                                containerSize="16px"
+                                emoji={wallet.emoji}
+                            />
+                            <Label2>{wallet.name || t('wallet_title')}</Label2>
+                        </WalletNameStyled>
+                    )}
+                </ListItemStyled>
+                <ListItemStyled hover={false}>
+                    <Body2>Recipients</Body2>
+                    <Label2>{form.rows.length}</Label2>
+                </ListItemStyled>
+                <ListItemStyled hover={false}>
+                    <Body2>List</Body2>
+                    <Label2>{listName}</Label2>
+                </ListItemStyled>
+                <ListItemStyled hover={false}>
+                    <Body2>Fee</Body2>
+                    <FeeContainer>
+                        {estimateError ? null : estimateLoading || !tonRate ? (
+                            <>
+                                <Skeleton margin="3px 0" width="100px" height="14px" />
+                                <Skeleton margin="2px 0" width="80px" height="12px" />
+                            </>
+                        ) : (
+                            <>
+                                <Label2>≈ {tonFee}</Label2>
+                                <Body3>{fiatFee}</Body3>
+                            </>
+                        )}
+                    </FeeContainer>
+                </ListItemStyled>
+            </ListBlockStyled>
+            <ButtonBlock
```

Comment:
<img width="817" alt="Screenshot 2024-04-29 at 3 00 22 PM" src="https://github.com/tonkeeper/tonkeeper-web/assets/22742808/2d285ac0-dbeb-4748-9d92-9b15eba8489f">

Could you please write an error with happen on this screen 
"The maximum number of transactions for Wallet Version 4 is four. If you need to send more transactions at one time, please use Wallet Version 5."

and "The maximum number of transactions for Wallet Version 5 is 255."

Reply:
Will do that in the Form Screen

Reply:
Done